public/js/validator.custom.js

```diff
@@ -0,0 +1,375 @@
+/*
+	Validator v1.0.5
+	(c) 2012 Yair Even Or <http://dropthebit.com>
+
+	MIT-style license.
+*/
+
+var validator = (function(){
+	var message, tests, checkField, validate, mark, unmark, field, minmax, defaults,
+		validateWords, lengthRange, lengthLimit, pattern, alertTxt, data,
+		email_illegalChars = /[\(\)\<\>\,\;\:\\\/\"\[\]]/,
+		email_filter = /^.+@.+\..{2,3}$/;
+
+	/* general text messages
+	*/
+	message = {
```

Comment:
@at-dungnh 
No, this is Gentelella package's validator, I just changed a little bit validate rules and customized some error messages. 
That blog is original.